### PR TITLE
add ES markdown version

### PR DESCRIPTION
**Pull request by Stephen-Gates**
Please check:

- Links (some go to English sites)
- emphasis on Must, Must Not, Should, May

### CODE REVIEW

File: od/2.1/es/index.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+layout: page
+---
+
+Versión: 2.1
+
+La Definición de Conocimiento Abierto aporta precisión al significado del término «abierto» (open) cuando se aplica al conocimiento, promueve un procomún robusto en el que cualquiera puede participar y maximiza su interoperabilidad.
+
+**Resumen:** *El conocimiento es abierto si cualquiera es libre de acceder a él, usarlo, modificarlo y compartirlo, estando sujeto a lo sumo a medidas que preserven su autoría y su apertura.*
+
+Este significado esencial se corresponde con el de «abierto» (open) cuando se refiere al software, como en la [Open Source Definition](http://opensource.org/docs/osd1), y es un sinónimo de «libre» (free) tal y como se usa en la [Definición de Software Libre](https://en.wikipedia.org/wiki/The_Free_Software_Definition) y en la [Definición de Obras Culturales libres](https://en.wikipedia.org/wiki/Definition_of_Free_Cultural_Works) .
```
Comment (mlinksva) [inline od/2.1/es/index.markdown]: The OSD link is broken. Maybe the trailing '1'? In any case I suggest using available eswp links:

- https://es.wikipedia.org/wiki/Open_Source_Definition
- https://es.wikipedia.org/wiki/Definici%C3%B3n_de_Software_Libre
- https://es.wikipedia.org/wiki/Definici%C3%B3n_de_obras_culturales_libres

Comment (Stephen-Gates) [reply od/2.1/es/index.markdown]: Done!

**Comment by Stephen-Gates** [comment]
So Travis says there's lots of broken links. This may be true but not in the code I changed ([test it](https://github.com/Stephen-Gates/opendefinition/blob/0ea739b8228fd421f756cc1c0dbd4b56c9a84df2/od/2.1/es/index.markdown)).

So how do all the links get corrected (if that's the problem)?

**Comment by mlinksva** [comment]
The broken tests are fine; there a tons of broken links throughout the site that no one PR will fix.

No further feedback at https://discuss.okfn.org/t/spanish-version-of-open-definition-2-1/5815/4 ... good to merge?

**Comment by Stephen-Gates** [comment]
Good from my perspective

**Pull request merged**